charts/prometheus-statsd-exporter/templates/ingress.yaml

```
@@ -0,0 +1,41 @@
+{{- if .Values.ingress.enabled -}}
+{{- $fullName := include "prometheus-statsd-exporter.fullname" . -}}
+{{- $svcPort := .Values.service.port -}}
+{{- if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion -}}
+apiVersion: networking.k8s.io/v1beta1
+{{- else -}}
+apiVersion: extensions/v1beta1
+{{- end }}
```

One more thing, we don't need this check if a prerequisite of this chart is
> Kubernetes 1.16+ with Beta APIs enabled

```suggestion
apiVersion: networking.k8s.io/v1beta1
```